AuxTel/AuxTel-Standard-Operations/AuxTel-Nighttime-Operations/Shutdown.rst

```diff
@@ -31,7 +31,7 @@ The Scheduler automatically adds new targets to the queue until the start of nau
 If there are no targets, the Scheduler will stop adding new ones.
```

Comment:
**Not able to add comment on line 17:** The parking position for AuxTel is ....
**Not able to add comment on line 19:** Duplicated phrase from line 16.
**Not able to add comment on line 24:** Pre-Conditions???